src/routes/TradingLeague/Entry/Points/Points.tsx

```diff
@@ -30,12 +30,10 @@ export const Points: React.FunctionComponent<PointsProps> = ({ points }) => {
   }
 
   return (
-    <CountUp start={0} end={points} delay={0} formattingFn={formatNumber}>
-      {({ countUpRef }) => (
-        <PointsTypography>
-          <span ref={countUpRef} />
-        </PointsTypography>
-      )}
-    </CountUp>
+    <PointsTypography>
+      <CountUp start={0} end={points} delay={0} formattingFn={formatNumber}>
```

Comment:
btw @ioanaroceanu2000 (unrelated to this PR, but reminded me of the points issue) i think there were some users that mentioned that points have appeared to go down over time which feels strange, my current guess is that the points are not calculated using the historical exchange rates of eth at the time that points are generated and rather use the most recent exchange rate

Reply:
@arturbeg oh, not good, I will look into it

Reply:
Seems unrelated to this :D

Reply:
yea:)

Reply:
thanks @ioanaroceanu2000, let's create a ticket and migrate this conversation to the qa channel on discord